src/libs/ReportActionsUtils.ts

```diff
@@ -1201,6 +1201,7 @@ function isOldDotReportAction(action: ReportAction | OldDotReportAction) {
         CONST.REPORT.ACTIONS.TYPE.EXPORTED_TO_QUICK_BOOKS,
         CONST.REPORT.ACTIONS.TYPE.REIMBURSEMENT_REQUESTED,
         CONST.REPORT.ACTIONS.TYPE.REIMBURSEMENT_SETUP,
+        CONST.REPORT.ACTIONS.TYPE.EXPORTED_TO_INTEGRATION,
```

Comment:
```suggestion
```

Let's just add another case to [getReportActionMessage](https://github.com/Expensify/App/blob/5cff64e5fc0410dfa0f08ed4b082ed62095ea834/src/libs/ReportUtils.ts#L3323) instead of making this an OldDot action.